shared/app/nav.native.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import CardStackTransitioner from 'react-navigation/src/views/CardStack/CardStackTransitioner'
 import GlobalError from './global-errors/container'
-import Offline from '../offline'
```

Comment:
cleanup the styles and use the fastBlank to reduce overdraw
fixes the issue where showing modals causes the entire stack to thrash